src/transaction/log_manager.c

```diff
@@ -1026,15 +1026,15 @@ log_unmount_active_file (THREAD_ENTRY * thread_p)
  */
 void
 log_initialize (THREAD_ENTRY * thread_p, const char *db_fullname, const char *logpath, const char *prefix_logname,
-		int ismedia_crash, BO_RESTART_ARG * r_args)
+		bool is_media_crash, BO_RESTART_ARG * r_args)
```

Comment:
Harmonized a bit parameter type.
In an upper scope it was bool; in an intermediate scope it was int; then, downwards, bool again.